charts/discovery-ui/values.yaml

```diff
@@ -20,7 +20,10 @@ env:
   api_host: "https://data.example.com"
   #contribute_host: "https://sharedata.example.com" #Optional location for an associated public Andi deployment
   gtm_id: "GTM-EXAMPLE"
+  #used in react_discovery_ui for websocket command
   base_url: "example.com"
+  #used in discovery_ui for header purposes
+  disc_ui_url: "example.com"
```

Comment:
Can we add on to this to describe the value itself more and less of it's use case

Ex:
```
# the domain where discovery ui will be deployed
# used in discovery_ui for header purposes 
disc_ui_url: "discovery.example.com"
```